content/en/docs/components/training/overview.md

```diff
@@ -1,65 +1,65 @@
 +++
 title = "Overview"
-description = "An overview of Training Operator"
+description = "An overview of the Training Operator"
 weight = 10
 +++
 
 {{% stable-status %}}
 
-## What is Training Operator ?
+## What is the Training Operator?
```

Comment:
I think, we can remove the question mark from the header, similar to this page: https://www.kubeflow.org/docs/started/introduction/#what-is-kubeflow